examples/basic-node/app.js

```diff
@@ -0,0 +1,12 @@
+var dust = require('dustjs-linkedin');
+
+var tmpl = dust.compile("Hello world! Using Dust version {version}!", "hello");
+dust.loadSource(tmpl);
```

Comment:
The other examples do pre-compile, but this was intended to be the most simple example possible.

How would you want pre-compilation to work? I didn't want to make them run `dustc`.